metrics/mixin/alerts/services-external.libsonnet

```diff
@@ -0,0 +1,27 @@
+{
+  prometheusAlerts+:: {
+    groups+: [
+      {
+        name: 'cluster-services-alert.rules',
+        rules: [
+          {
+            alert: 'ClusterObjectStoreState',
+            expr: |||
+              ocs_rgw_health_status{%(ocsExporterSelector)s} > 1
+            ||| % $._config,
+            'for': $._config.clusterObjectStoreStateAlertTime,
+            labels: {
+              severity: 'critical',
+            },
+            annotations: {
+              message: 'Cluster Object Store is in unhealthy state. Please check Ceph cluster health or RGW connection.',
+              description: 'Cluster Object Store is in unhealthy state for more than %s. Please check Ceph cluster health or RGW connection.' % $._config.clusterObjectStoreStateAlertTime,
+              storage_type: $._config.objectStorageType,
+              severity_level: 'error',
```

Comment:
We can get rid of this.

Reply:
Yeah we can, but let's keep all the alerts related to OCS same. These will differ from the rook ones then, might create confusion.